adapters/discord/src/types/application.ts

```diff
@@ -38,6 +39,21 @@ export interface Application {
   cover_image?: string
   /** the application's public flags */
   flags?: integer
+  /** up to 5 tags describing the content and functionality of the application */
+  tags?: [string, string?, string?, string?, string?]
+  /** settings for the application's default in-app authorization link, if enabled */
+  install_params?: InstallParams
+  /**	the application's default custom authorization link, if enabled */
```

Comment:
```suggestion
  /** the application's default custom authorization link, if enabled */
```

Replace tabs with spaces (with more occurences below)